pkg/crds/calico/policy.networking.k8s.io_adminnetworkpolicies.yaml

```diff
@@ -4,7 +4,7 @@ metadata:
   annotations:
     api-approved.kubernetes.io: https://github.com/kubernetes-sigs/network-policy-api/pull/30
     policy.networking.k8s.io/bundle-version: v0.1.1
-    policy.networking.k8s.io/channel: standard
+    policy.networking.k8s.io/channel: experimental
```

Comment:
It seems the wrong direction to change this to experimental when standard was here before, though I guess we haven't make AdminNetworkPolicies available before so I guess it is ok for this value to change. I assume this has been reviewed in Calico also.